ios/Classes/TiMapCustomAnnotationView.m

```diff
@@ -36,7 +36,7 @@ - (void)setProxy:(TiViewProxy*)customView
 
 - (void)initWithProxy:(TiViewProxy*)customView
 {
-    theProxy = [customView retain];
+    theProxy = customView;
```

Comment:
And it actually breaks after zooming in and out a few times.
<img width="854" alt="screen shot 2016-11-15 at 14 49 08" src="https://cloud.githubusercontent.com/assets/10667698/20327630/bac50b26-ab42-11e6-8a3c-5d3fff6c8a80.png">